docs/references.rst

```diff
@@ -0,0 +1,166 @@
+==========
+References
+==========
+
+.. contents:: Table of Contents
+    :depth: 2
+
+
+--------------------
+
+
+:guilabel:`GameEvent`
+=====================
+
+- **Inherits:** `Node <https://docs.godotengine.org/en/stable/classes/class_node.html>`_
+- **Inherited by:** None
+- **Singleton**
+
+This class contains global signals that can be emitted and connected by any node.
+
+
++---------------------------+---------+
+| Signals                   | Methods |
++===========================+=========+
+|| :term:`CTF_capture_flag` ||        |
+|| :term:`CTF_return_flag`  ||        |
+|| :term:`increase_score`   ||        |
++---------------------------+---------+
+
+
+.. topic:: Signals
+
+    .. glossary::
+        CTF_capture_flag
+            [CTF] Emitted when a team capture the enemy's flag.
+
+            **Parameters**
+
+            - team_id (`int <https://docs.godotengine.org/en/stable/getting_started/scripting/gdscript/gdscript_basics.html#int>`_)
+
+        CTF_return_flag
+            [CTF] Emitted when a flag is returned to its station.
```

Comment:
Maybe you should say "to its team's station" to make it more obvious what the team_id param is for